htdocs/install/mysql/tables/llx_societe_accountancy.sql

```diff
@@ -0,0 +1,28 @@
+-- ========================================================================
+-- Copyright (C) 2021		Open-Dsi	<support@open-dsi.fr>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <http://www.gnu.org/licenses/>.
+--
+-- ========================================================================
+
+create table llx_societe_accountancy
```

Comment:
Same use a table
llx_societe_perentity
In a future, we may have more fields than accountancy